lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -1051,10 +1058,17 @@ int mme_app_handle_create_sess_resp(
     OAILOG_FUNC_RETURN(LOG_MME_APP, RETURNerror);
   }
 
+  OAILOG_INFO_UE(
+      LOG_MME_APP, ue_context_p->emm_context._imsi64,
+      "Received S11_CREATE_SESSION_RESPONSE from S+P-GW for ue "
+      "id " MME_UE_S1AP_ID_FMT "\n",
+      ue_context_p->mme_ue_s1ap_id);
+
   OAILOG_DEBUG_UE(
       LOG_MME_APP, ue_context_p->emm_context._imsi64,
-      "MME S11 teid = %u, cause = %d, ue_id = %u\n", create_sess_resp_pP->teid,
-      create_sess_resp_pP->cause.cause_value, ue_context_p->mme_ue_s1ap_id);
+      "MME S11 teid = %u, cause = %d, ue_id = " MME_UE_S1AP_ID_FMT "\n",
+      create_sess_resp_pP->teid, create_sess_resp_pP->cause.cause_value,
+      ue_context_p->mme_ue_s1ap_id);
```

Comment:
why don't we combine these?

Reply:
+1 on this, let's use INFO and combine the information logged here